src/styles/_buttons.scss

```diff
@@ -0,0 +1,27 @@
+
+.btn-dark-theme {
+	display: flex;
+	justify-content: flex-end;   
+	padding: 5px 25px;
+	border-radius: $buttonBorderRadius;
+	font-weight: bold;
+	align-self: flex-end;
+	transition: 0.5s all;
```

Comment:
could we research and, if necessary, create constants for animations?

Reply:
for a moment its the only animation